configs/watch_pending.cfg

```diff
@@ -16,15 +16,21 @@
         "^b2g_try_(emulator|emulator-jb|emulator-jb|emulator-l|emulator-kk)(-debug)?_(dep|nightly|nonunified)": "try-linux64",
         "^b2g_(?!try)\\S+_linux(32|64)_gecko(-debug)?": "bld-linux64",
         "^Firefox (mozilla-central|mozilla-aurora|gum) linux(64)? l10n nightly": "bld-linux64",
+        "^Firefox \\S+ win(32|64) l10n nightly": "b-2008",
         "^Thunderbird (comm-central|comm-aurora) linux(64)? l10n nightly": "bld-linux64",
+        "^Thunderbird \\S+ win(32|64) l10n nightly": "b-2008",
         "^graphene_(?!try)\\S+_linux(32|64)(-debug)?": "bld-linux64",
+        "^graphene_(?!try)\\S+_win(32|64)": "b-2008",
+        "^graphene_try\\S+_win(32|64)": "try-2008",
         "^horizon_(?!try)\\S+_linux(32|64)(-debug)?": "bld-linux64",
+        "^(TB )?WINNT (5.2|6.1|6.2)( x86-64)?(.*(?! try)).* (pgo-)?build": "b-2008",
+        "^(TB )?WINNT (5.2|6.1|6.2)( x86-64)?.* nightly": "b-2008",
+        "^(TB )?WINNT (5.2|6.1|6.2)( x86-64)? try": "try-2008",
         "^b2g_try_linux(32|64)_gecko(-debug)?": "try-linux64",
         "^Ubuntu VM 12.04 (?!x64).*": "tst-linux32",
         "^Ubuntu (Code Coverage )?VM 12.04 x64.*": "tst-linux64",
         "^Ubuntu Mulet VM 12.04 x64.*": "tst-linux64",
         "^Ubuntu ASAN VM 12.04 x64.*": "tst-linux64",
-        "^Ubuntu TSAN VM 12.04 x64.*": "tst-linux64",
```

Comment:
reverted removal (arr: I don't think there were any matches, but it doesn't hurt to leave it in)